aggregator_core/Cargo.toml

```diff
@@ -40,25 +40,25 @@ janus_messages.workspace = true
 k8s-openapi.workspace = true
 kube.workspace = true
 opentelemetry.workspace = true
-postgres-protocol = "0.6.6"
-postgres-types = { version = "0.2.6", features = ["derive", "array-impls"] }
+postgres-protocol = { workspace = true }
+postgres-types = { workspace = true, features = ["derive", "array-impls"] }
```

Comment:
Darn, it doesn't look like `cargo-autoinherit` factors out common features--both usages of `postgres-types` use the same featureset.